migrations/initdb-banners.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable new-cap */
+
+'use strict';
+
+const owner = process.env.DATASTORE_SEQUELIZE_OWNER || 'postgres';
+const schema = process.env.DATASTORE_SEQUELIZE_SCHEMA || 'public';
+const lockTimeout = process.env.DATASTORE_SEQUELIZE_LOCKTIMEOUT || '2s';
+const prefix = process.env.DATASTORE_SEQUELIZE_PREFIX || '';
+const table = `${prefix}banners`;
+
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        await queryInterface.sequelize.transaction(async (transaction) => {
+            await queryInterface.sequelize.query(
+                `SET lock_timeout TO '${lockTimeout}';`, { transaction });
+            await queryInterface.sequelize.query(
+                `SET ROLE TO ${owner};`, { transaction });
```

Comment:
`SET ROLE` is available only from MySQL 8.0.
MySQL 5.7, which is still supported and widely used, isn't support this operation.

Reply:
similar to lock_timeout will remove this, preferred ROLE to block DROP statements. A better option will be to remove the down operation.